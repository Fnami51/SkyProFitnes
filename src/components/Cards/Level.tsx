import React from 'react' // нужно для тестов

function Level({ value }: { value: number }) {
	const heights = [3, 6, 9, 11, 14]

	return (
		<div className='flex gap-[0.75px] items-end h-[18px] w-[18px] p-[2px]'>
			{heights.map((height, i) => (
				<div
					key={i}
					className={`w-[2.25px] rounded-[50px] ${
						i < value ? 'bg-[#00C1FF]' : 'bg-[#D9D9D9]'
					}`}
					style={{ height: `${height}px` }}
				/>
			))}
		</div>
	)
}

export default Level
