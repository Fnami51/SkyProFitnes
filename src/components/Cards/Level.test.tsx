import React from 'react' // нужно для тестов
import { render } from '@testing-library/react'
import Level from './Level'

test('Level matches snapshot', () => {
	const { asFragment } = render(<Level value={3} />)
	expect(asFragment()).toMatchSnapshot()
})
