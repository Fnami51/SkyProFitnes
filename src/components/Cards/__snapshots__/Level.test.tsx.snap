// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Level matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex gap-[0.75px] items-end h-[18px] w-[18px] p-[2px]"
  >
    <div
      class="w-[2.25px] rounded-[50px] bg-[#00C1FF]"
      style="height: 3px;"
    />
    <div
      class="w-[2.25px] rounded-[50px] bg-[#00C1FF]"
      style="height: 5.75px;"
    />
    <div
      class="w-[2.25px] rounded-[50px] bg-[#00C1FF]"
      style="height: 8.5px;"
    />
    <div
      class="w-[2.25px] rounded-[50px] bg-[#D9D9D9]"
      style="height: 11.25px;"
    />
    <div
      class="w-[2.25px] rounded-[50px] bg-[#D9D9D9]"
      style="height: 14px;"
    />
  </div>
</DocumentFragment>
`;
